folly/File.cpp

```diff
@@ -120,8 +120,10 @@ File File::dupCloseOnExec() const {
     int fd;
 #ifdef _WIN32
     fd = ::dup(fd_);
-#else
+#elseif defined(F_DUPFD_CLOEXEC)
```

Comment:
`#elseif` doesn't exist, did you mean `#elif`?

Reply:
Thank you for noticing, will fix this. (`elseif` crept in from Tcl.)

Reply:
Fixed.